### PR TITLE
Add generic config flow block

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

This PR makes a start with adding some generic re-usable blocks for commons things.
In this case, the config flow instructions. Most of them are the same, or copied from each other, but still differ a tiny bit caused by small changes to the UI lately.

This also help with implementing the new my portal, as we have central block to adjust.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_includes/integrations/config_flow.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{% capture name %}{{ include.name | default: page.title }}{% endcapture %}
+
+## Configuration
```
Comment (frenck) [inline source/_includes/integrations/config_flow.md:3]: So, I'm a bit in doubt if I should add this heading in here or not...

File: source/_integrations/airly.markdown [hunk 2]
```diff
@@ -15,15 +15,13 @@ ha_platforms:
...
+The Airly integration uses the [Airly](https://airly.eu/) web service as a source for air quality data for your location.
 
 ## Setup
 
 To generate an Airly API key, go to [Airly for developers](https://developer.airly.eu/register) page.
 
-## Configuration
-
-To add Airly to your installation, go to **Configuration** >> **Integrations** in the UI, click the button with `+` sign and from the list of integrations select **Airly**. By default, the values will be taken from the Home Assistant configuration.
+{% include integrations/config_flow.md %}
```
Comment (frenck) [inline source/_integrations/airly.markdown:24]: This an example on why I think having the header in that template, makes this a bit more unclear in Markdown.

File: source/_includes/integrations/config_flow.md (hunk 1 above)
Comment (ludeeus) [reply source/_includes/integrations/config_flow.md:3]: I think there should be a heading, but maybe "Setup" or something along those lines.

Comment (frenck) [reply source/_includes/integrations/config_flow.md:3]: The name of the heading was not the concern at this point. Its called `Configuration` everywhere right now 😉 

can always be changed.

File: source/_integrations/airly.markdown (hunk 2 above)
Comment (ludeeus) [reply source/_integrations/airly.markdown:24]: ah, right...

```
{% if include.hide_header != true %}

## Configuration

{% endif %}
```

And then:
`{% include integrations/config_flow.md hide_header=true %}`

**Comment by frenck** [comment]
Marking it a draft, migrating a bunch first.

File: source/_integrations/homematicip_cloud.markdown [hunk 3]
```diff
@@ -41,19 +41,8 @@ There is currently support for the following device types within Home Assistant:
...
-Fill the form:
-
-* Your **access point ID** (SGTIN)
-* Optional a **name** to identify your access point, this will be used to prefix your device names.
-* The **PIN**, mandatory if one is used in the native HomematicIP App.
-
-The authentification token will be generated and stored internally.
-
-## Setup the integration using the configuration files
+{% include integrations/config_flow.md %}
```
Comment (ludeeus) [inline source/_integrations/homematicip_cloud.markdown:44]:
```suggestion
{% include integrations/config_flow.md %}

```

File: source/_integrations/hvv_departures.markdown [hunk 4]
```diff
@@ -16,18 +16,9 @@ ha_platforms:
 
 The `hvv_departures` sensor will display the departures of busses, trains and ferries in Hamburg.
 
-## Configuration
+{% include integrations/config_flow.md %}
```
Comment (ludeeus) [inline source/_integrations/hvv_departures.markdown:19]:
```suggestion
{% include integrations/config_flow.md %}

```

File: source/_integrations/izone.markdown [hunk 5]
```diff
@@ -20,16 +20,13 @@ The `iZone` integration allows access of control of a local [iZone](https://izon
 
 Any current iZone unit with ducted reverse cycle airconditioning and the CB wired or wireless bridge device installed should currently work. There is currently no support for the iZone lights, reticulation, or other devices.
 
-## Configuration
+{% include integrations/config_flow.md %}
```
Comment (ludeeus) [inline source/_integrations/izone.markdown:23]:
```suggestion
{% include integrations/config_flow.md %}

```

File: source/_integrations/metoffice.markdown [hunk 6]
```diff
@@ -16,9 +16,9 @@ ha_platforms:
 
 The `metoffice` weather platform uses the Met Office's [DataPoint API](https://www.metoffice.gov.uk/datapoint) for weather data.
 
-## Configuration
+{% include integrations/config_flow.md %}
```
Comment (ludeeus) [inline source/_integrations/metoffice.markdown:19]:
```suggestion
{% include integrations/config_flow.md %}

```

File: source/_integrations/upb.markdown [hunk 7]
```diff
@@ -18,12 +18,11 @@ The UPB integration allows Home Assistant to connect to a Universal Powerline Bu
...
 - Scene
 
-## Configuration
-
-To add UPB to your installation, go to **Configuration** >> **Integrations** in the UI, click the button with `+` sign and from the list of integrations select **Universal Powerline Bus (UPB)**.
+## Prerequisites
 
 The UPB integration requires that an export from the `UPStart` UPB configuration program. To create an export, in `UPStart`, click the UPB button in the top left and select **Export to File**. This will create a file with the `.upe` extension. The file must be placed in the configuration directory of your Home Assistant installation.
 
+{% include integrations/config_flow.md %}
```
Comment (ludeeus) [inline source/_integrations/upb.markdown:25]:
```suggestion
{% include integrations/config_flow.md %}

```

File: source/_integrations/vesync.markdown [hunk 8]
```diff
@@ -46,27 +46,7 @@ This integration supports devices controllable by the VeSync App.  The following
 
 - LEVOIT Smart Wifi Air Purifier (LV-PUR131S)
 
-## Configuration
-
-To use this integration, all devices must be registered with the VeSync App. Once registration is complete, you can add the VeSync integration by adding the VeSync integration in the configuration section of the frontend and entering your username and password.  You can also use the traditional configuration method by adding the following to your `configuration.yaml` file:
```
Comment (ludeeus) [inline source/_integrations/vesync.markdown:51]: > To use this integration, all devices must be registered with the VeSync App.

I don't think this line should be removed, it's a prerequisite to use the integration.

**Review by ludeeus — DISMISSED** [review]
Great job @frenck :medal_sports:

File: source/_integrations/vesync.markdown (hunk 8 above)
Comment (frenck) [reply source/_integrations/vesync.markdown:51]: Tomorrow 😓 💤

**Pull request merged**